lib/AccessSystem/TelegramBot.pm

```diff
@@ -217,6 +237,10 @@ sub read_message ($self, $message) {
         resend_inductions => {
             match => qr{^/resend_inductions\b},
             help => '"/resend_inductions" (for yourself) or "/resend_inductions Fred Bloggs"'
+        },
+        door_log      => {
+            match => qr{^/door_log\b},
+            help => 'Display the last 5 users of the door (directors-only).'
```

Comment:
I'm not sure what's the structure of the usage log, but the `door_log` takes `10` rows, this says `the last 5 users`. Is that correct?